### PR TITLE
add contributing guidelines to README

### PR DESCRIPTION
**Pull request by japhir**

### CODE REVIEW

**Comment by japhir** [comment]
Should solve #5, but may be further improved

File: README.org [hunk 1]
```diff
@@ -125,6 +125,14 @@ If you have the Chemacs2 configuration cloned to =~/.emacs.d= or =~/.config/emac
 
 Then launch it with =emacs --with-profile rational=!
 
+* Contributing
+[[http://makeapullrequest.com][https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square]]
+[[https://github.com/bbatsov/emacs-lisp-style-guide][https://img.shields.io/badge/elisp-style%20guide-purple?style=flat-square]]
+
+This is a community-run modular Emacs configuration, for which we appreciate feedback (in the form of issues) and pull requests. Feel free to open an issue prior to opening a pull request if you're not certain your idea is in the spirit of the [[file:~/Downloads/rational-emacs/README.org::*Principles][Principles]].
+
+If you enjoy crafting your computing experience, join the [[https://systemcrafters.cc/][SystemCrafters]] community!
```
Comment (daviwil) [inline README.org:134]:
```suggestion
If you enjoy crafting your computing experience, join the [[https://systemcrafters.net/][SystemCrafters]] community!
```

File: README.org [hunk 2]
```diff
@@ -125,6 +125,14 @@ If you have the Chemacs2 configuration cloned to =~/.emacs.d= or =~/.config/emac
 
 Then launch it with =emacs --with-profile rational=!
 
+* Contributing
+[[http://makeapullrequest.com][https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square]]
+[[https://github.com/bbatsov/emacs-lisp-style-guide][https://img.shields.io/badge/elisp-style%20guide-purple?style=flat-square]]
+
+This is a community-run modular Emacs configuration, for which we appreciate feedback (in the form of issues) and pull requests. Feel free to open an issue prior to opening a pull request if you're not certain your idea is in the spirit of the [[file:~/Downloads/rational-emacs/README.org::*Principles][Principles]].
```
Comment (daviwil) [inline README.org:132]:
```suggestion
This is a community-run modular Emacs configuration, for which we appreciate feedback in the form of issues and pull requests. Feel free to open an issue prior to opening a pull request if you're not certain your idea is in the spirit of the [[file:~/Downloads/rational-emacs/README.org::*Principles][Principles]].
```

**Review by daviwil — COMMENTED** [review]
Looks great, thanks a lot for adding this! Just a couple of suggestions and we can get this merged.

**Comment by japhir** [comment]
> Looks great, thanks a lot for adding this! Just a couple of suggestions and we can get this merged.

Note that I got the URL from the GitHub description, maybe update that as well?

**Comment by japhir** [comment]
Should be ready for merge now :)

**Comment by daviwil** [comment]
Awesome, thanks!

**Comment by japhir** [comment]
Aww I just noticed that the second badge is still not working and neither is the link to the principles ☹️

**Pull request merged**